site/docs/reference/Connectors/capture-connectors/google-sheets.md

```diff
@@ -0,0 +1,103 @@
+the service account key method is the only supported method using the command line.
+
+### Prerequisites for OAuth
+
+:::caution Beta
+OAuth implementation is under active development and is coming soon.
+Use the service account key method for now.
+:::
+
+* A link to a Google spreadsheet. Simply copy the link from your browser.
+
+* Your Google account username and password.
+
+### Prerequisites using a service account key
+
+* A link to a Google spreadsheet. Simply copy the link from your browser.
+
+* Google Sheets and Google Drive APIs enabled on your Google account.
+
+* A Google service account with:
+  * A JSON key generated.
+  * Access to the source spreadsheet.
+
+Follow the steps below to meet these prerequisites:
+
+1. [Enable](https://support.google.com/googleapi/answer/6158841?hl=en) the Google Sheets and Google Drive APIs
+for the Google [project](https://cloud.google.com/storage/docs/projects) with which your spreadsheet is associated.
+(Unless you actively develop with Google Cloud, you'll likely just have one option).
+
+2. Create a [service account and generate a JSON key](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount).
+During setup, grant the account the **Viewer** role on your project.
+You'll copy the contents of the downloaded key file into the Service Account Credentials parameter when you configure the connector.
+
+3. Share your Google spreadsheet with the service account. You may either share the sheet so that anyone with the link can view it,
+or share explicitly with the service account's email address.
+
+## Configuration
+
+You configure connectors either in the Flow web app, or by directly editing the catalog spec YAML.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors.
+The values and YAML sample below provide configuration details specific to the Google Sheets source connector.
+
+### Properties
+
+#### Endpoint
+
+The following properties reflect the Service Account Key authentication method.
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/credentials`** | Credentials | Google API Credentials for connecting to Google Sheets and Google Drive APIs | object | Required |
+| **`/credentials/auth_type`** | Authentication Type | Authentication method. Set to `Service`. | string | Required |
+| **`credentials/service_account_info`** | Service Account Credentials | Contents of the JSON key file generated during setup. | string | Required |
+| **`/spreadsheet_id`** | Spreadsheet Link | The link to your spreadsheet. | string | Required |
+
+#### Bindings
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/stream`** | Sheet | Each sheet in your Google Sheets document. | string | Required |
+| **`/syncMode`** | Sync mode | Connection method. Always set to `incremental`. | string | Required |
```

Comment:
Google sheets only supports `full_refresh`, so it should always be set to `full_refresh`.
```suggestion
| **`/syncMode`** | Sync mode | Connection method. Always set to `full_refresh`. | string | Required |
```